base/initdefs.jl

```diff
@@ -18,10 +18,27 @@ An array of the command line arguments passed to Julia, as strings.
 """
 const ARGS = String[]
 
+"""
+    exit([code])
+
+Quit (or control-D at the prompt). The default exit code is zero, indicating that the
+processes completed successfully.
+"""
 exit(n) = ccall(:jl_exit, Void, (Int32,), n)
 exit() = exit(0)
+
+"""
+    quit()
+
+Quit the program indicating that the processes completed successfully. This function calls
+`exit(0)` (see [`exit`](@ref)).
+"""
 quit() = exit()
 
+# hint for people who forget parentheses
```

Comment:
This is one of the rare cases where you might want to overload `display`:
```jl
Base.display(d::Base.REPL.REPLDisplay, ::typeof(exit)) = display(d, Text("Use exit(), quit(), or Ctrl-D to exit"))
```
will work and should only affect output in the REPL.